acs-category-theory-notes.tex

```diff
@@ -81,51 +114,74 @@
 \newcommand{\union}{\cup}
 
 \newcommand{\nelem}[1]{ \mathbf{ #1 } }
+\newcommand{\id}[1]{ \mathrm{id}_{ #1 } }
+\newcommand{\nats}{\mathbb{N}}
 
 \begin{document}
 
-\maketitle
+\begin{center} {\huge \sc
+Category Theory, Type Theory, and Logic\\
+  Lecture Notes\\[4mm]}
+  \Large Marcelo Fiore
+\end{center}
+
+
+\begin{multicols}{3}[\section*{Contributors}]
+Dhruv Makwana\\ 
+Shaun Steenkamp\\
+Andrej Ivašković\\
+Oliver Richardson
+\end{multicols}
+\clearpage
+
 
 \tableofcontents
 
 \newpage
-\section{Lecture 1 (October 5): How to think Categorically}
-\vspace*{-5mm}\hspace*{7.75mm}
-M Fiore, D Makwana, S Steenkamp
-\bigskip
+\chapter{How to think Categorically}
+\lecturedetails{5 October 2017}{M Fiore, D Makwana, S Steenkamp, O Richardson}
 
 Introduction to Category Theory with applications to Logic and Type Theory.
 
 \emph{Not a spectator sport.}
 
-Approach: Find an understanding somewhere between the concrete (examples) and
-the abstract (``nonsense'').
```

Comment:
Please contribute each lecture in it's own branch.
Changes to earlier lectures and stylistic changes should be separate branches.